apps/sms/views/conversation/js/conversation.js

```diff
@@ -1468,6 +1468,17 @@ var ConversationView = {
     this._stopRenderingNextStep = true;
   },
 
+  /**
```

Comment:
__nit (:bike: ):__ It seems would be better to have method description as well (more in description and less for return statement description)